pkg/monitor/monitor/monitor_manager.go

```diff
@@ -700,3 +668,80 @@ func (m *MonitorManager) syncAssetSecret(monitor *v1alpha1.TidbMonitor, store *S
 	}
 	return nil
 }
+
+func (m *MonitorManager) syncDashboardMetricStorage(tc *v1alpha1.TidbCluster, tm *v1alpha1.TidbMonitor) error {
+	if tc.Spec.PD == nil {
+		return nil
+	}
+	pdEtcdClient, err := m.deps.PDControl.GetPDEtcdClient(pdapi.Namespace(tc.Namespace), tc.Name, tc.IsTLSClusterEnabled())
+
+	if err != nil {
+		return err
+	}
+	if tc.IsTLSClusterEnabled() {
```

Comment:
no need to check `tc.IsTLSClusterEnabled()` after #3929. see https://github.com/pingcap/tidb-operator/pull/3929/files#diff-a8fc0b3952619cae74a70fd53685f27e760ccb52a03682eff264c51f5fefdc47R116